invenio_communities/invitations/services/request_types.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 Northwestern University.
+#
+# Invenio-Communities is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invitation request types."""
+
+from flask_babelex import lazy_gettext as _
+from invenio_requests.customizations import RequestState, BaseRequestType
+
+from .actions import AcceptAction
+
+
+class CommunityMemberInvitation(BaseRequestType):
+    """Community member invitation request type."""
+
+    type_id = 'community-member-invitation'
+    name = _('Community Member Invitation')
```

Comment:
This will be displayed in the request detail page so go with either 

- Invitation
- Community invitation
- Member invitation

Also only [first letter in first word in capital](https://inveniordm.docs.cern.ch/maintenance/styleguide/#captilization)

![Screenshot 2022-02-02 at 11 05 00](https://user-images.githubusercontent.com/1698163/152133146-f3099e7d-a34b-48cb-815a-5d5d0ea32e73.png)

Reply:
What will we call it when we invite records to be part of the community? I was thinking a parallel naming structure:

- Community member invitation
- Community record invitation

I've also been prefacing the names with "Community", so that they are namespaced. Member / Record... is very generic and would require namespacing by other modules down the road anyway.